basic/1.hashes/README.md

```diff
@@ -0,0 +1,41 @@
+# 1. Entries Exercise
```

Comment:
This should be changed to hashes right?

Reply:
fixed all references to entries exercise in README